### PR TITLE
[WFLY-9885] elytron security configuration is ignored in RA connectio…

### PR DESCRIPTION
**Pull request by simkam**
…n definition

added missing elytron configuration options to condition that determines if security is defined for connection definition + updated tests

https://issues.jboss.org/browse/WFLY-9885

@maeste  @fl4via please review

### CODE REVIEW

**Review by maeste — APPROVED** [review]

**Comment by simkam** [comment]
retest this please

File: testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/jca/rar/MultipleManagedConnectionFactoryWithSubjectVerification.java [hunk 1]
```diff
@@ -0,0 +1,52 @@
+/*
+ *
+ * Copyright 2017 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
```
Comment (stuartwdouglas) [inline testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/jca/rar/MultipleManagedConnectionFactoryWithSubjectVerification.java]: This should be the lesser GPL

**Comment by simkam** [comment]
licence header updated

**Comment by fl4via** [comment]
Retest this please

**Comment by fl4via** [comment]
@simkam LGTM

**Pull request merged**